src/igvfd/schemas/institutional_certificate.json

```diff
@@ -0,0 +1,138 @@
+{
+    "title": "Institutional Certificate",
+    "description": "An institutional certificate defining the data sharing limitations for data authorized for submission to the IGVF portal.",
+    "$id": "/profiles/institutional_certificate.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "certificate_identifier",
+        "controlled_access",
+        "urls"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "certificate_identifier",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "dependentSchemas": {},
```

Comment:
i think we need a dependency to require "data_use_limitation" if the controlled access=true